repo/stable/index.yaml

```diff
@@ -2,11 +2,19 @@ apiVersion: v1
 entries:
   guestbook:
   - apiVersion: v1
-    created: 2018-05-10T21:50:41.837568406-04:00
+    created: 2019-02-21T14:09:27.965528851Z
     description: A Helm chart to deploy Guestbook three tier web application.
-    digest: 25da3e4d75a0e0e51426fc53b72f1ee0769a8f00e776a621e2205cffbfe4dc98
+    digest: 74d7c81b83aeea22feb3c4710f86c42b5be61e3783da5fb649edd66f3df55f90
     name: guestbook
     urls:
-    - https://raw.githubusercontent.com/ibm/helm101/master/repo/stable/guestbook-0.1.0.tgz
+    - https://ibm.github.io/helm101/guestbook-0.2.0.tgz
+    version: 0.2.0
+  - apiVersion: v1
+    created: 2019-02-21T14:09:27.964727056Z
+    description: A Helm chart to deploy Guestbook three tier web application.
+    digest: 7f186687f43008ed68e9e4a09aa4e45586e2d6ed5f85287c64bda3f5a7ea4d12
+    name: guestbook
+    urls:
+    - https://ibm.github.io/helm101/guestbook-0.1.0.tgz
     version: 0.1.0
-generated: 2018-05-10T21:50:41.83648523-04:00
+generated: 2019-02-21T14:09:27.909109099Z
```

Comment:
Why we need to change anything in the 0.1.0 section?

Reply:
This is when `helm index` runs its updates the index file to add the new chart version (0.2.0) and changes the generated timestamp accordingly. I think this probably should changes as it has been regenerated.

I have updated changed the "created" timestamp for chart v0.1.0 back to its original.